plugins/GSdx/GSCrc.cpp

```diff
@@ -395,7 +403,7 @@ CRC::Game CRC::m_games[] =
 	{0x79C952B0, DevilMayCry3, JP, 0}, // SE
 	{0x7F3DDEAB, DevilMayCry3, JP, 0},
 	{0x05931990, DevilMayCry3, KO, 0},
-	{0x4AD36D59, DevilMayCry3, RU, 0},
+	{0x4AD36D59, DevilMayCry3, RU, 0}, // same CRC as RU
```

Comment:
It's already RU region.

Reply:
And what about the line {0x8E265148, TombRaiderUnderworld, RU, 0}?
In the RU version, CRC-hacks are not applied to it, and glitches are not fixed.

Reply:
That one is fine just change the first letter of the code comment to use capital letter.

Reply:
That's right?

Reply:
I don't see anything different other than the PR title.

You need to make the changes in the file and push them.